src/mono/dbi/cordb-code.cpp

```diff
@@ -0,0 +1,150 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+// File: CORDB-CODE.CPP
+//
+
+#include <cordb-blocking-obj.h>
+#include <cordb-breakpoint.h>
+#include <cordb-code.h>
+#include <cordb-function.h>
+#include <cordb-process.h>
+#include <cordb.h>
+
+using namespace std;
+
+CordbCode::CordbCode(Connection* conn, CordbFunction* func) : CordbBaseMono(conn)
+{
+    this->m_pFunction = func;
+    m_nSize = -1;
+}
+
+HRESULT CordbCode::IsIL(BOOL* pbIL)
+{
+    LOG((LF_CORDB, LL_INFO100000, "CordbCode - IsIL - NOT IMPLEMENTED\n"));
+    return E_NOTIMPL;
+}
+
+HRESULT CordbCode::GetFunction(ICorDebugFunction** ppFunction)
+{
+    LOG((LF_CORDB, LL_INFO100000, "CordbCode - GetFunction - NOT IMPLEMENTED\n"));
+    return E_NOTIMPL;
+}
+
+HRESULT CordbCode::GetAddress(CORDB_ADDRESS* pStart)
+{
+    LOG((LF_CORDB, LL_INFO100000, "CordbCode - GetAddress - NOT IMPLEMENTED\n"));
+    return E_NOTIMPL;
+}
+
+ULONG32 CordbCode::GetSize() {
+    if (m_nSize == -1) {
```

Comment:
I would invert the `if` to return early. also fix the `{` new line
```cpp
    if (m_nSize != -1)
    {
          return m_nSize;
    }
```

Reply:
Done